wiki/markdown/curl-examples.md

```diff
@@ -86,6 +82,26 @@ It is possible to provide both username and password directly in the request, as
 
 More information and examples can be found in the [EI back-end documentation](https://github.com/eiffel-community/eiffel-intelligence/blob/master/wiki/markdown/authentication.md)
 
+## <a id="status" /> /status
+
+<table>
+    <tr>
+        <th>Context Path</th>
```

Comment:
Context-path is wrong to used.
Context-path is the url/path suffix that comes between host address and the entrypoints.
Should be named entrypoint or something other.
More info about context path in Url addresses.
https://www.eclipse.org/jetty/documentation/9.4.x/configuring-contexts.html

Reply:
See that we have this other places as well,, should be updated with a more correct name/type in all places.

Reply:
Done.